packages/simple-hub/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+const baseConfig = require('../../.eslintrc.js');
```

Comment:
Like some other configs, you don't need to use `require` to inherit from the monorepo root config file. You can add `../../.eslintrc` to `extends` of the exported object.

Reply:
Good call. 84842e51